components/generic/RelatedCollections.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-container
+  <div
```

Comment:
Looks like this is squishing the related collections on the search page to the right side.
Example: /en/search?query="Mario Testino"

Reply:
ok fixed this